From 36d78f8ca27e7fb24343d6922bfc0511d1ad84ce Mon Sep 17 00:00:00 2001
From: Bruno Haible <bruno@clisp.org>
Date: Mon, 8 Sep 2025 01:00:32 +0200
Subject: [PATCH] Improve efficiency of "git clone".

---
 bootstrap-funclib.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bootstrap-funclib.sh b/bootstrap-funclib.sh
index ba531140..ae459fdb 100644
--- a/bootstrap-funclib.sh
+++ b/bootstrap-funclib.sh
@@ -35,7 +35,7 @@ export LC_ALL
 # Honor $PERL, but work even if there is none.
 PERL="${PERL-perl}"
 
-default_gnulib_url=https://git.savannah.gnu.org/git/gnulib.git
+default_gnulib_url=https://https.git.savannah.gnu.org/git/gnulib.git
 
 # Copyright year, for the --version output.
 copyright_year=`echo "$scriptlibversion" | sed -e 's/[^0-9].*//'`
-- 
2.50.1

